boards/arm/v2m_beetle/Kconfig.defconfig

```diff
@@ -8,10 +8,6 @@ if BOARD_V2M_BEETLE
 config BOARD
 	default "v2m_beetle"
 
-config PINMUX_BEETLE
```

Comment:
wondering if the pinmux driver should be deleted then...

Reply:
oh, there's a Kconfig.beetle... yeah I think that should go as well